[Bug]: IT failiures

### Describe the bug

It seems that the project is using OpenSearch `1.3.9` all ITs are failing:

```
self = <it.TestCluster object at 0x1076a3890>, distribution_version = '1.3.9', node_name = 'metrics-store', provision_config_instance = 'defaults', http_port = 10200

    def install(self, distribution_version, node_name, provision_config_instance, http_port):
        self.http_port = http_port
        transport_port = http_port + 100
        try:
            err, retcode = process.run_subprocess_with_stderr(
                "opensearch-benchmark install --configuration-name={cfg} --distribution-version={dist} --build-type=tar "
                "--http-port={http_port} --node={node_name} --master-nodes="
                "{node_name} --provision-config-instance={provision_config_instance} "
                "--seed-hosts=\"127.0.0.1:{transport_port}\"".format(cfg=self.cfg,
                                                                     dist=distribution_version,
                                                                     http_port=http_port,
                                                                     node_name=node_name,
                                                                     provision_config_instance=provision_config_instance,
                                                                     transport_port=transport_port))
            if retcode != 0:
                raise AssertionError("Failed to install OpenSearch {}.".format(distribution_version), err)
            self.installation_id = json.loads(err)["installation-id"]
        except BaseException as e:
>           raise AssertionError("Failed to install OpenSearch {}.".format(distribution_version), e)
E           AssertionError: ('Failed to install OpenSearch 1.3.9.', AssertionError('Failed to install OpenSearch 1.3.9.', "[ERROR] Cannot install. Install a JDK with one of the versions [17, 16, 15, 14, 13, 12, 11, 8] and point to it with one of ['JAVA17_HOME', 'JAVA16_HOME', 'JAVA15_HOME', 'JAVA14_HOME', 'JAVA13_HOME', 'JAVA12_HOME', 'JAVA11_HOME', 'JAVA8_HOME', 'JAVA_HOME'].\n"))


```

### To reproduce

`make it`

### Expected behavior

_No response_

### Screenshots

If applicable, add screenshots to help explain your problem.

### Host / Environment

_No response_

### Additional context

_No response_

### Relevant log output

```shell

```

--- a/it/__init__.py
+++ b/it/__init__.py
@@ -37,7 +37,7 @@ from osbenchmark import client, config, version, paths
 from osbenchmark.utils import process
 
 CONFIG_NAMES = ["in-memory-it", "os-it"]
-DISTRIBUTIONS = ["1.3.9", "2.5.0"]
+DISTRIBUTIONS = ["1.3.9", "2.17.1"]
 WORKLOADS = ["geonames", "nyc_taxis", "http_logs", "nested"]
 BASE_COMMANDS = ["opensearch-benchmark", "osb"]
 ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))

